diary-alexander-jaffray/modernized/32-ch-16-scotland.adoc

```diff
@@ -329,8 +348,13 @@ the spiritual welfare of which, we have been engaged in contemplating;
 he is at length about to put in force, by patient resignation unto death,
 that injunction of the Apostle, "`We ought to lay down our lives for the brethren.`" John 3:16.
 
+[.embedded-content-document.letter]
```

Comment:
So here begins a letter that has letters within. I did what I thought looked good, but idk if it's right! It seems that John Bowstead wrote a letter to Francis Stamper about Peter Gardiner, and inserted a couple of Peter's letters. At least that's what I could make of it. :)